src/misc/unicode.cpp

```diff
@@ -594,7 +596,7 @@ static std::optional<uint8_t> grapheme_to_screen_code(const Grapheme& grapheme,
 // Conversion routines
 // ***************************************************************************
 
-static bool utf8_to_wide(const std::string& str_in, std::vector<uint16_t>& str_out)
+static wide_string utf8_to_wide(const std::string& str_in)
```

Comment:
Just `str` instead of `str_in` (all `const` args are inputs, that's obvious).

Reply:
OK, changed everywhere.